src/packagedcode/rubygems.py

```diff
@@ -647,93 +648,133 @@ def normalize(gem_data, known_fields=known_fields):
     )
 
 
-def parse_spec(location):
-    pass
+# defaults fields of .gemspec file
+data_dic = {
+    "name": None,
+    "version": None,
+    "authors": None,
+    "email": None,
+    "summary": None,
+    "description": None,
+    "homepage": None,
+    "license": None,
+    "dependencies": None
+    }
 
 
-class GemSpec(object):
+def parse_spec(location):
+    """
+    Returns a dictionary which containing .gemspec file data.
+    """
+    file_data = []
+    with io.open(location, "r", encoding="utf-8") as f:
+        file_data = [line.rstrip('\n') for line in f]
+
+    dependencies = []
+
+    for individual in file_data:
+        # update the value of name
+        if 'name' in individual:
+            if '"' in individual:
+                name = individual.split('"')
+            elif "'" in individual:
+                name = individual.split("'")
```

Comment:
What is this doing, exactly?

Reply:
In some files string is like `spec.name = "name"` and in some files it is `spec.name = 'name'`.

This if-elif help in recognizing in `"` and `'`

Reply:
But is this really needed?

Reply:
@majurg In our .json output file, the output is like `"name": "<name>"`. If we will not replace `"` or `'` the output will be like `"name: ""<name>""`
What I am actually doing is extracting the data between the `""` or `''`

I think this can be done in more efficient way:
```python
name = individual.split('= ')
name = name[1].strip('\'"')
data_dic.update({'name' : name})
```

What do you say??

Reply:
Yes you are right here, I forget gemspec files are actual ruby code.